github-management/subproject-site-requests.md

```diff
@@ -0,0 +1,161 @@
+  - [Example Subproject Domain Request](#example-subproject-domain-request)
+- [Processing a Netlify Site Request](#processing-a-netlify-site-request)
+
+
+## Requesting a Netlify Site
+
+
+- Create an issue in the [kubernetes/org] repository using the
+  [Netlify Site Request Template]. It will ask for:
+  - Requesting sub-project and associated information. Linking to an issue or
+    other supporting material for the Netlify request associated with the
+    subproject is encouraged and will speed processing of the request.
+  - Desired domain name. It should follow the pattern of
+    `<sub-project>.sigs.k8s.io` and match the
+    [request](#requesting-a-subproject-domain) for a subproject domain.
+
+- A member of the GitHub Admin team handling Netlify requests will respond and
+  follow up with any questions in the issue.
+
+- Once Netlify has been configured, [request a subproject
+  domain](#subproject-domain-request) and reference the issue you created. You
+  may use the Netlify site URL (example: `kubernetes-sigs-foo.netlify.com`) till
+  the DNS request has been processed.
+
+- Update the project to include a [site specific][site-config] configuration
+  file at the root of the repository ([`netlify.toml`][site-config]). Below is a
+  [Hugo] based example that you may use as a reference.
+
+
+Once complete, the site should be accessible.
+
+
+### Example Netlify Configuration
+
+```toml
+[build]
+base = "site/"
+publish = "site/public/"
+command = "hugo"
+
+[build.environment]
+HUGO_VERSION = "0.53"
+
+[context.production.environment]
+HUGO_ENV = "production"
+HUGO_BASEURL = "https://foo.sigs.k8s.io/"
+
+[context.deploy-preview]
+command = "hugo --enableGitInfo --buildFuture -b $DEPLOY_PRIME_URL"
+
+[context.branch-deploy]
+command = "hugo --enableGitInfo --buildFuture -b $DEPLOY_PRIME_URL"
+  ```
+
+## Subproject Domain Request
+
+Subproject domains may be requested for a Kubernetes Org managed Netlify site in
+the form of: `<project>.sigs.k8s.io`.
+
+To do so, create an issue using the [DNS Update Request] issue template in the
+[kubernetes/k8s.io] repository. Use the below guidelines for your
+[DNS Update Request].
+
+- **Type of DNS update:** Create
+- **Domain being modified:** `k8s.io`
+- **New DNS Record:**
+  ```yaml
+  # https://github.com/kubernetes-sigs/<project> (<contact or owners>)
+  <project>.sigs:
+    type: CNAME
+    value: <netlify url>
+  ```
+- **Reason for update:** Provide a description and link to the Netlify request
+  issue made in [kubernetes/org].
+
+
+### Example Subproject Domain Request
+
+````
+### Type of DNS update:
+Create
+
+### Domain being modified:
+k8s.io
+
+### New DNS Record:
+```yaml
+# https://github.com/kubernetes-sigs/foo (@bar)
+foo.sigs:
+  type: CNAME
+  value: kubernetes-sigs-foo.netlify.com
+  ```
+### Reason for update:
+The sig-foo subproject has requested a netlify site to host its documentation. ref #0000, #0000.
+````
+
+
+## Processing a Netlify Site Request
```

Comment:
This is the part that would be done by the github owners, yes? If so, can we just make a `**NOTE:**` as such?